.github/workflows/release.yml

```
@@ -3,162 +3,27 @@ name: Release
+      - name: Install Yarn
+        run: npm install -g yarn
+      - uses: grafana/plugin-actions/build-plugin@main
+        with:
+          # see https://grafana.com/developers/plugin-tools/publish-a-plugin/sign-a-plugin#generate-an-access-policy-token to generate it
+          # save the value in your repository secrets
+          grafana_token: ${{ secrets.GRAFANA_ACCESS_POLICY_TOKEN }}
+          go-version: "1.24.1"
+          node-version: "22.x"
+          use_changelog_generator: true
+          # creates a signed build provenance attestation to verify the authenticity of the plugin build
+          attestation: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)